### PR TITLE
10 topic page

### PR DESCRIPTION
**Pull request by MFStevenson**

### CODE REVIEW

File: src/pages/CommentsPage.jsx [hunk 1]
```diff
@@ -1,27 +1,44 @@
...
+  const [err, setErr] = useState(null);
+
+  const updateInput = (e) => {
+    setInput(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const postBody = { username: "jessjelly", comment: input };
+    postComment(article_id, postBody)
```
Comment (popatre) [inline src/pages/CommentsPage.jsx:21]: Did you branch from a branch?

File: src/pages/CommentsPage.jsx [hunk 2]
```diff
@@ -34,16 +51,33 @@ const CommentsPage = () => {
...
-      <CommentForm />
-      <button onClick={handleBackToArticle}>Back to article</button>
+
+      <section id="comment-form">
+        <p>Post a comment</p>
+        {err ? <p>{err}</p> : null}
+        <form onSubmit={handleSubmit}>
+          <label>
+            Comment{" "}
+            <input
```
Comment (popatre) [inline src/pages/CommentsPage.jsx:69]: I think you've branched from a branch - im not sure why this is in this commit. Think you're going to have some tricky merge conflicts

File: src/utils/api.js [hunk 3]
```diff
@@ -24,6 +24,10 @@ export const patchArticleVotes = (article_id, votes) => {
   return ncNewsApi.patch(`/articles/${article_id}`, { inc_votes: votes });
 };
 
-export const postComment = (article_id) => {
-  return ncNewsApi.post(`/api/articles/${article_id}/comments`);
+export const postComment = (article_id, postBody) => {
```
Comment (popatre) [inline src/utils/api.js:27]: Its really difficult to check this ticket as it seems to have loads of stuff to do with other tickets as well

**Review by popatre — CHANGES_REQUESTED** [review]
This PR was really hard to look at as it seems to have lots of stuff committed from other open PRs. 

Topic PR wise - how can i send someone all the articles to do with football? At the moment, they have to go to one page, then click a button. 
For example, heres all the business news articles from the BBC - https://www.bbc.co.uk/news/business

You can go straight there and i dont need to tell you to go one place, then find a button to click. 

 Because of the current set up, they disappear when you refresh the page as well which isnt great UI/UX

**Review by nc-modou — APPROVED** [review]
Good response to Jonathan's comments! Topics now has it's own page which works great, and the change to the url means you can now access them by URL directly. Feel free to merge!

**Pull request merged**